assets/js/components/KeyMetrics/ChipTabGroup/index.js

```diff
@@ -253,6 +283,12 @@ export default function ChipTabGroup( { allMetricItems, savedItemSlugs } ) {
 										<CheckMark width={ 12 } height={ 12 } />
 									</span>
 								) }
+								{ KEY_METRICS_SUGGESTED_GROUP_SLUG ===
```

Comment:
Let's update the current selection icon to use this approach as well.

Reply:
Did you push your changes? I don't see it has been updated.

Reply:
@eugene-manuilov I totally misunderstood the change here and thought it was a slug refactor 😄 I have now refactored it to use the same approach taken in Chip.js